ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/ClientTest.java

```diff
@@ -1390,6 +1391,15 @@ public void shouldDescribeSource() throws Exception {
     assertThat(description.getSourceConstraints(), hasItems("s1", "s2"));
   }
 
+  @Test
+  public void shouldGetServerInfo() throws Exception {
+    final ServerInfo serverInfo = javaClient.serverInfo().get();
+    assertThat(serverInfo.getVersion(), is(AppInfo.getVersion()));
+    assertThat(serverInfo.getKsqlServiceId(), is("ksql-service-id"));
+    assertThat(serverInfo.getKafkaClusterId(), is("kafka-cluster-id"));
+    assertThat(serverInfo.getServerStatus(), is("server-status"));
+  }
+
```

Comment:
Let's also add a quick integration test to `ClientIntegrationTest.java`, to protect against someone inadvertently updating the API format without updating the client.